pkg/tests/end_to_end_tests/continuous_agg_test.go

```diff
@@ -13,9 +13,12 @@ import (
 	"github.com/timescale/promscale/pkg/internal/testhelpers"
```

Comment:
I think we need some tests to make sure that  the top-level `execute_data_retention_policy` does the right thing with views and that `set_metric_retention_period` works on caggs as well.